code/datums/mood_events/needs_events.dm

```diff
@@ -91,3 +91,40 @@
 	description = "There's nothing like the feeling of a freshly laundered jumpsuit."
 	mood_change = 2
 	timeout = 10 MINUTES
+
+
+//nutrition
+/datum/mood_event/ate_event
+	timeout = 10 MINUTES
+
+/datum/mood_event/ate_event/no_table
+	description = "I had to eat a meal off the ground. Can't we get a table around here?"
+	mood_change = -5
+
+/datum/mood_event/ate_event/no_chair
+	description = "I had to eat a meal while standing. Can't we get some chairs around here?"
+	mood_change = -5
+
+/datum/mood_event/ate_event/no_service
+	description = "I had to eat a meal in a place like this. I wish I was eating at the bar or diner!"
+	mood_change = -5
+
+/datum/mood_event/ate_event/no_utensils
+	description = "I had to eat a meal with my bare hands. Can't we get some forks around here?"
+	mood_change = -5
+
+/datum/mood_event/ate_event/table
```

Comment:
+20 mood bonus for 10 minutes invalidates the system, lower to at most +2

Reply:
done